src/Engine/Conditions.cpp

```diff
@@ -19,9 +19,9 @@ MM_DECLARE_FLAGS(ConditionFlags, ConditionFlag)
 MM_DECLARE_OPERATORS_FOR_FLAGS(ConditionFlags)
 
 struct ConditionTableEntry {
-    ConditionFlags flags;
-    ItemEnchantment enchantment = ITEM_ENCHANTMENT_NULL;
-    std::array<ConditionEquipment, 3> equipment = {{}};
+    ConditionFlags _flags;
```

Comment:
Style-wise if it's a struct with public members we don't add "_" to the fields. "_" is for private members.